cli_tools/gce_onestep_image_import/main.go

```diff
@@ -0,0 +1,84 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// GCE one-step image import tool
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging/service"
+	onestepImporter "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_onestep_image_import/onestep_importer"
+)
+
+func main() {
+	log.SetPrefix("[onestep-import-image] ")
+	// 1. Parse flags
+	importerArgs, err := onestepImporter.NewImportArguments(os.Args[1:])
+	if err != nil {
+		log.Println(err)
+		os.Exit(1)
+	}
+
+	importEntry := func() (service.Loggable, error) {
+		return onestepImporter.Run(importerArgs)
+	}
+
+	// 2. Run Onestep Importer
+	project := importerArgs.Project
```

Comment:
this one doesn't work. The Project will be updated in validate&populate step, and the updated value should be reflected in RunWithServerLogging.
1. need to pass a "*ImportArguments".
2. don't assign "project := importerArgs.Project". Just pass &(importArgs.Project) to RunWithServerLogging.

You see the existing ImageImport did "project := importerArgs.Project", it's because its validate&populate has been done before this line.

Reply:
got it, thanks!